Gemfile

```
@@ -117,6 +117,7 @@ end
 
 group :test do
   gem 'capybara', '~> 3.38'
+  gem 'climate_control'
```

Might be worth adding a little description, even though this hasn't been done elsewhere yet
```suggestion

  # Used to mock ENV values in specs
  gem 'climate_control'

```